week1/HomeWork/index.js

```diff
@@ -0,0 +1,32 @@
+let http = require('http');
+const port = 3000;
+var state = 10; 
+
+let server = http.createServer( (request, response) => {
+
+   if(request.url === "/state") {
+	   response.write(String(state));
+   }
+   else if(request.url === "/add") {
+	   state++;
+	   response.write(String(state));
+   }
+   else if(request.url === "/remove") {
+	   state--;
+	   response.write(String(state));
+   }
+   else if(request.url === "/reset") {
+	   state = 10;
+	   response.write(String(state));
+   }
+   else{
+	   response.write("Not found! , Please check the url path");
+   }
+   
+   response.end()
```

Comment:
I notice that you return without declaring a 'Content-Type' header. In this case it's probably fine, but if it were, for example, JSON or HTML, we're leaving it up to the browser to try and figure out what the content was supposed to be. Can you add a Content-Type header, to explicitly declare what kind of content you're returning?